ts/main.ts

```diff
@@ -0,0 +1,64 @@
+interface Anime {
+  mal_id: number;
+  title: string;
+  episodes: number;
+  images: {
+    jpg: {
+      image_url: string;
+    };
+  };
+  [key: string]: any; // To account for any additional fields
+}
```

Comment:
If you aren't going to use any other fields then this wouldn't be necessary since you are asserting the type of data. You just won't be able to access the other properties if they aren't assigned in the interface.